configuration/device-tree.md

```diff
@@ -405,56 +479,58 @@ Here are some examples of different types of properties, with parameters to modi
-#### 2.2.7: Further overlay examples
+Sometime, however, it is very useful to be able to create a label with one overlay and use it from another. Firmware released since Feb 14 2020 has the ability to declare some labels as being global - the `__export__` node:
+```
+	...
+	public: ...
 
-There is a growing collection of overlay source files hosted in the Raspberry Pi/Linux GitHub repository [here](https://github.com/raspberrypi/linux/tree/rpi-4.4.y/arch/arm/boot/dts/overlays).
+	__exports__ {
+		public; // Export the label 'public' to the base DT
+	};
+};
+```
+When this overlay is applied, the loader strips out all symbols except those that have been exported, in this case `public`, and rewrites the path to make it relative to the target of the fragment containing the label. Overlays loaded after this one can then refer to `&public`.
 
-<a name="part3"></a>
 ## Part 3: Using Device Trees on Raspberry Pi
+<a name="part3"></a>
 
+### 3.1: DTBs, overlays and config.txt
 <a name="part3.1"></a>
-### 3.1: Overlays and config.txt
 
-On a Raspberry Pi it is the job of the loader (one of the `start.elf` images) to combine overlays with an appropriate base device tree, and then to pass a fully resolved Device Tree to the kernel. The base Device Trees are located alongside `start.elf` in the FAT partition (/boot from Linux), named `bcm2708-rpi-b.dtb`, `bcm2708-rpi-b-plus.dtb`, `bcm2708-rpi-cm.dtb`, and `bcm2709-rpi-2-b.dtb`. Note that Models A and A+ will use the "b" and "b-plus" variants, respectively. This selection is automatic, and allows the same SD card image to be used in a variety of devices.
+On a Raspberry Pi it is the job of the loader (one of the `start.elf` images) to combine overlays with an appropriate base device tree, and then to pass a fully resolved Device Tree to the kernel. The base Device Trees are located alongside `start.elf` in the FAT partition (/boot from Linux), named `bcm2711-rpi-4-b.dtb`, `bcm2710-rpi-3-b-plus.dtb`, etc. Note that some models (3A+, A, A+) will use the "b" equivalents (3B+, B, B+), respectively. This selection is automatic, and allows the same SD card image to be used in a variety of devices.
 
-Note that DT and ATAGs are mutually exclusive. As a result, passing a DT blob to a kernel that doesn't understand it causes a boot failure. To guard against this, the loader checks kernel images for DT-compatibility, which is marked by a trailer added by the `mkknlimg` utility; this can be found in the `scripts` directory of a recent kernel source tree. Any kernel without a trailer is assumed to be non-DT-capable.
+Note that DT and ATAGs are mutually exclusive, and passing a DT blob to a kernel that doesn't understand it causes a boot failure. The firmware will always try to load the DT and pass it to the kernel, since all kernels since rpi-4.4.y will not function without a DTB. You can override this by adding `device_tree=` in config.txt, forcing the use of ATAGs, which can be useful for simple "bare-metal" kernels.
```

Comment:
"doesn't understand it **will cause** a boot failure"?

Reply:
That sounds better.